module_4/gift_certificates_system/repository/src/test/java/com/epam/esm/repository/configuration/RepositoryConfigurationTest.java

```diff
@@ -0,0 +1,48 @@
+package com.epam.esm.repository.configuration;
+
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.MessageSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.support.ResourceBundleMessageSource;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+import org.springframework.validation.beanvalidation.MethodValidationPostProcessor;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import java.util.Locale;
+
+/**
+ * Main ESM application configuration.
+ */
+@Configuration
+@EnableTransactionManagement
+@EnableJpaRepositories("com.epam.esm")
+@ComponentScan("com.epam.esm")
+@EntityScan("com.epam.esm")
+@SpringBootConfiguration
+public class RepositoryConfigurationTest implements WebMvcConfigurer {
```

Comment:
Incorrect naming, looks as if it is test for configuration, not test configuration

Reply:
Thanks, was [renamed](https://github.com/n199a/epam_training_center/pull/4/commits/7284879fec13d1ad2e6683c55327916e6fbad158).